packages/fxa-content-server/app/scripts/views/settings.js

```diff
@@ -175,7 +175,11 @@ const View = BaseView.extend({
 
   _setupAvatarChangeLinks() {
     if (this.supportsAvatarUpload()) {
-      this.$('.avatar-wrapper > *').wrap(
+      const avatarImage = this.$('.avatar-wrapper > img');
+      const avatarElement = avatarImage.length
+        ? avatarImage
+        : this.$('.avatar-wrapper > span');
+      avatarElement.wrap(
```

Comment:
Since we have the spinner displayed in the settings view now, this check for what element to wrap became necessary. `> *` was proving a little buggy, as `.change-avatar` was in some cases appending twice.

`.avatar-wrapper > span` is for when the default image is showing.